src/Core/Analysis/AnalysisIssue.cs

```diff
@@ -38,6 +42,7 @@ public class AnalysisIssue : IAnalysisIssue
             EndLineOffset = endLineOffset;
             FilePath = filePath;
             Message = message;
+            Locations = locations.Reverse().ToList().AsReadOnly();
```

Comment:
There should be a doc comment on the interface describing the ordering of the locations.
It also seems weird that this class is the one that's responsible for reversing them, rather than the converter class that is mapping from whatever is the underlying source of the issues.

Reply:
Yea I was debating about that too. Originally wanted to re-use with the deamon, but the deamon has no support for secondary locations in the protobuf, so it's not relevant. 
I'll move it to the cfamilyHelper.